From e8ec492c1f16328149e2e2c1eed6c300529bcb7a Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@ubuntu.com>
Date: Tue, 4 Sep 2012 23:35:38 +0100
Subject: [PATCH 025/364] * Makefile.am (EXTRA_DIST): Add linguas.sh.  It's
 only strictly required for checkouts from bzr, but it may be useful for users
 or distributors wishing to update translations against a tarball
 distribution, and it can be helpful for the tarball to be a superset of
 what's in bzr.

---
 ChangeLog   | 8 ++++++++
 Makefile.am | 2 ++
 2 files changed, 10 insertions(+)

diff --git a/ChangeLog b/ChangeLog
index b1950ab..fbef0c3 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,5 +1,13 @@
 2012-09-04  Colin Watson  <cjwatson@ubuntu.com>
 
+	* Makefile.am (EXTRA_DIST): Add linguas.sh.  It's only strictly
+	required for checkouts from bzr, but it may be useful for users or
+	distributors wishing to update translations against a tarball
+	distribution, and it can be helpful for the tarball to be a superset
+	of what's in bzr.
+
+2012-09-04  Colin Watson  <cjwatson@ubuntu.com>
+
 	* Makefile.am (EXTRA_DIST): Add
 	grub-core/tests/boot/linux.init-mips.S,
 	grub-core/tests/boot/linux.init-ppc.S, and
diff --git a/Makefile.am b/Makefile.am
index 3bb911c..30aa5a7 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -390,3 +390,5 @@ SUCCESSFUL_BOOT_STRING=3e49994fd5d82b7c9298d672d774080d
 BOOTCHECK_TIMEOUT=180
 
 bootcheck: $(BOOTCHECKS)
+
+EXTRA_DIST += linguas.sh
-- 
1.8.1.4

